applications/ShapeOptimizationApplication/custom_utilities/geometry_utilities.h

```diff
@@ -312,6 +318,147 @@ class GeometryUtilities
         return length;
     }
 
+// --------------------------------------------------------------------------
+    double ComputeVolume()
```

Comment:
please split into header/source

Reply:
done

Reply:
This is why python stuff should only be used in the custom_python folder

Initially it was ok doing it like this, but for a long time we have split the apps into Core/Py-Interface
One reason is to compile the core (of the app) once and then the py-interface with different versions of python
I think this is for example used for the wheels as it brings down significantly the compilation time 

as I said fine for now but should be improved in the long run

Reply:
#9513